cfme/tests/configure/test_timeprofile.py

```diff
@@ -100,13 +89,8 @@ def test_timeprofile_description_required_error_validation():
         description=None,
         scope='Current User',
         timezone="(GMT-10:00) Hawaii")
-    if version.current_version() > "5.7":
-        tp.create(cancel=True)
-        assert tp.timeprofile_form.description.angular_help_block == "Required"
-        if version.current_version() > "5.8":
-            assert form_buttons.save.is_dimmed
-        else:
-            assert form_buttons.add.is_dimmed
-    else:
-        with error.expected("Description is required"):
-            tp.create()
+    view = navigate_to(st.Timeprofile, 'All')
+    tp.create(cancel=True)
+    if view.timeprofile_form.description.help_block == "Required":
```

Comment:
Ans here.

Reply:
Here I am using a help_block property of the Input widget. So can't make this change.